libs/librepcb/project/boards/items/bi_netpoint.h

```diff
@@ -75,6 +75,7 @@ class BI_NetPoint final : public BI_Base,
   // General Methods
   void addToBoard() override;
   void removeFromBoard() override;
+  UnsignedLength distance(Point destination);
```

Comment:
According our naming conventions, the method should be named `getDistance()`, or `getDistanceTo()` to be more expressive. But actually to be honest, I'd simply remove this method since it's trivial for callers to use `getPosition()` instead and calculate the distance to the other point "manually" since it's trivial :wink: